earth_enterprise/src/SConstruct

```diff
@@ -367,6 +367,37 @@ installdirs = {
     'htdocs': InstServerDir('gehttpd/htdocs'),
 }
 
+def get_cc_version(cc_path):
+  """Extracts the version number of a C/C++ compiler (like GCC) as a list of
+  strings."""
+
+  process = subprocess.Popen(
+    [cc_path, '--version'], stdout=subprocess.PIPE)
```

Comment:
Looks like this will need to be changed to `cc_dir`.

Reply:
its the compiler path, so leaving it as cc_path